xml/System.Numerics/Vector`1.xml

```diff
@@ -48,7 +48,20 @@
  <xref:System.Numerics.Vector%601> is an immutable structure that represents a single vector of a specified numeric type. The count of a <xref:System.Numerics.Vector%601> instance is fixed, but its upper limit is CPU-register dependent. It is intended to be used as a building block for vectorizing large algorithms.  
   
  The <xref:System.Numerics.Vector%601> structure provides support for hardware acceleration.  
-  
+ 
+ The following table shows which primitive numeric data type and operation combination uses intrinsic instructions for faster executions:
```

Comment:
this table doesn't render well:
![image](https://user-images.githubusercontent.com/12971179/46654560-fbb2db80-cb5d-11e8-882a-55bd5b1ce1f5.png)

Maybe try using a MarkDown table instead?

| data type | + | - | \* | / |
|------------|---|--|--|--|
|sbyte|Yes|Yes|No|No|

For example:
```
| data type | + | - | \* | / |
|------------|---|--|--|--|
|sbyte|Yes|Yes|No|No|
```

Reply:
Copied the full table from the corefx issue:
```
| | `+` | `-` | `*` | `/` |
| --- | :---: | :---: | :---: | :---: |
|`sbyte` | Yes | Yes | No | No |
|`byte` | Yes | Yes | No | No |
|`short` | Yes | Yes | Yes | No |
|`ushort` | Yes | Yes | No | No |
| `int` | Yes | Yes | Yes | No |
| `uint` | Yes | Yes | No | No |
| `long` | Yes | Yes | No | No |
| `ulong` | Yes | Yes | No | No |
| `float` | Yes | Yes | Yes | Yes |
| `double` | Yes | Yes | Yes | Yes |
```

| | `+` | `-` | `*` | `/` |
| --- | :---: | :---: | :---: | :---: |
|`sbyte` | Yes | Yes | No | No |
|`byte` | Yes | Yes | No | No |
|`short` | Yes | Yes | Yes | No |
|`ushort` | Yes | Yes | No | No |
| `int` | Yes | Yes | Yes | No |
| `uint` | Yes | Yes | No | No |
| `long` | Yes | Yes | No | No |
| `ulong` | Yes | Yes | No | No |
| `float` | Yes | Yes | Yes | Yes |
| `double` | Yes | Yes | Yes | Yes |

Reply:
Seems I did something wrong when I copied it. I hope it is fixed now.